src/app/blog/[slug]/page.tsx

```diff
@@ -0,0 +1,97 @@
+import { Fragment } from "react";
```

Comment:
nit: a cool updated React thing is that we can use `<></>` instead of `<Fragment></Fragment>`